src/dashboard/src/components/charts/allOrgDonut/defaultData.ts

```diff
@@ -1,28 +1,28 @@
 // Data for the Donut Chart with specified colors
 export const defaultData = {
-  space: '0 TB',
-  used: '0 TB',
-  available: '0 TB',
+  space: '10 TB',
```

Comment:
Having fake data here will result in this information displayed when it shouldn't.  Currently the default data is a base I use when displaying the component.

We should come up with a way for you use fake data until you can get Docker running.

Reply:
Removed the fake data.  If I need to see it I can just add/undo